### PR TITLE
Add entry point access and rename test file

### PR DESCRIPTION
**Pull request by jennmald**
The entry point code is fixed and lines for testing were removed. I took into account the renaming of the test file. Commented is the website for entry point information. Closed the previous pull request.

### CODE REVIEW

**Review by mrakitin — APPROVED** [review]

**Comment by mrakitin** [comment]
There are a few complaints from flake8 which have to be fixed before the PR is merged:
```
$ flake8
./sirepo_bluesky/__init__.py:5:1: E402 module level import not at top of file
./sirepo_bluesky/__init__.py:11:1: W191 indentation contains tabs
./sirepo_bluesky/__init__.py:12:1: W191 indentation contains tabs
./sirepo_bluesky/tests/test_entrypoint.py:3:1: E302 expected 2 blank lines, found 1
./sirepo_bluesky/tests/test_entrypoint.py:4:4: E111 indentation is not a multiple of     
./sirepo_bluesky/tests/test_entrypoint.py:4:4: F841 local variable 'db' is assigned to but never used
The command "flake8" exited with 1.
```

**Comment by jennmald** [comment]
@jklynch and I are working on it, thank you!

**Comment by mrakitin** [comment]
Thanks, @jennmald.

It makes sense to apply the `black` formatting the same way as it's done in https://github.com/bluesky/bluesky-queueserver. I think it's a bit larger project, and should be done in a separate PR. I'll create a separate issue for that.

File: sirepo_bluesky/__init__.py [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
 from ._version import get_versions
 __version__ = get_versions()['version']
 del get_versions
+
+# Look up a driver class by its name in registry.
+catalog_class = intake.registry["bluesky-mongo-normalized-catalog"]
+
+sirepo_bluesky_catalog_instance = catalog_class(
+        metadatastore_db="mongodb://localhost:27017/md",
+        asset_registry_db="mongodb://localhosty:27017/ar",
```
Comment (mrakitin) [inline sirepo_bluesky/__init__.py:11]: I actually like the name - `localhosty` :)
```suggestion
        asset_registry_db="mongodb://localhost:27017/ar",
```

**Comment by mrakitin** [comment]
That concerns me in tests:
```
E       AttributeError: module 'intake' has no attribute 'cat'
```

Should we use a specific intake version?

**Comment by jklynch** [comment]
@danielballan would you take a look at this mysterious intake [failure](https://travis-ci.org/github/NSLS-II/sirepo-bluesky/jobs/767927244#L827)? It only happened on python 3.6.

**Comment by mrakitin** [comment]
I think something has changed between intake [v0.6.1](https://github.com/intake/intake/blob/5f285278d5256521d20962b152e160e4241d9824/setup.py#L66) and [v0.6.2](https://github.com/intake/intake/blob/8acc70d9adb19344ca15dee948315828b61e87b2/setup.py#L67), where Python 3.7 became a minimum required version. In the test with Python 3.6 the v0.6.1 was pulled in. I think we can skip the Python 3.6 tests in this project from now on.

**Comment by mrakitin** [comment]
@jennmald, @jklynch, I have an update for CI via https://github.com/jennmald/sirepo-bluesky/pull/2, please check/merge. There is an upstream issue https://github.com/radiasoft/sirepo/issues/3528 regarding the failed tests. I hope we will figure out how to properly add the `testuser`'s simulation dirs soon.

**Comment by jklynch** [comment]
@mrakitin Thanks for noticing intake v0.6.2 does not support python 3.6. I looked at the `classifiers` but not `python_requires`. 😞

**Comment by mrakitin** [comment]
@jklynch, @jennmald, here is another PR with the fix: https://github.com/jennmald/sirepo-bluesky/pull/3. Then we are good to merge this one.

**Comment by jennmald** [comment]
Thank you for all the help @mrakitin !

**Pull request merged**